crates/cext/src/transpiler/target.rs

```diff
@@ -869,6 +872,474 @@ pub unsafe extern "C" fn qk_target_num_instructions(target: *const Target) -> us
     target.len()
 }
 
+/// @ingroup QkTarget
+/// Return the index in which an operation is located based on its name.
+///
+/// @param target A pointer to the ``QkTarget``.
+/// @param name A pointer to the name string which we will look up an index for.
+///
+/// # Example
+/// ```c
+///     QkTarget *target = qk_target_new(5);
+///     QkTargetEntry *target_entry = qk_target_entry_new(QkGate_H);
+///     qk_target_add_instruction(target, target_entry);
+///
+///     size_t op_idx = qk_target_op_get_index(target, "h");
+/// ```
+///
+/// # Safety
+///
+/// Behavior is undefined if ``QkTarget`` is not a valid, non-null pointer to a ``QkTarget``.
+/// Behavior is undefined if ``name`` is not a pointer to a valid null-terminated string.
+#[unsafe(no_mangle)]
+#[cfg(feature = "cbinding")]
+pub unsafe extern "C" fn qk_target_op_get_index(
```

Comment:
IMO, the ordering of name components should be a little different, maybe `qk_target_get_op_index`, since we're asking the target to find an index, not asking an op what its index is.

Reply:
``qk_target_get_op_index`` can be a valid choice too, since I do want to mention that the index maps to an operation.